### PR TITLE
Fix: tooltip

### PR DESCRIPTION
**Pull request by doichev-kostia**
FIRELANE 🔥🔥🔥

### CODE REVIEW

File: stories/tooltip/index.jsx [hunk 1]
```diff
@@ -8,28 +8,39 @@ import { WrappedComponent } from '../helpers/wrapped';
...
-          <Tooltip content="Tooltip message will show up here">TOP</Tooltip>
-        </Col>
-        <Col s="2" m="4" l="3">
-          <Tooltip content="Tooltip message will show up here." position="right">
+      <Row style={ {
+        justifyContent: 'space-around',
+        marginBottom: '1.5rem',
+        marginTop: '1rem',
+      } }>
+        <Col s='6' m='6' l='6'>
```
Comment (lazzystep) [inline stories/tooltip/index.jsx:16]: wrong breakpoints (I know that you just formatted it, but can you make it right here?)
correct format: `xs="6" sm="6" lg="6"`

File: stories/tooltip/index.jsx [hunk 2]
```diff
@@ -8,28 +8,39 @@ import { WrappedComponent } from '../helpers/wrapped';
...
+          <Tooltip arrow content='Tooltip message will show up here.'
+                   position='right'>
             RIGHT
           </Tooltip>
         </Col>
+        <Col s='6' m='6' l='6' style={{marginLeft: '1rem'}}>
```
Comment (lazzystep) [inline stories/tooltip/index.jsx:22]: wrong breakpoints (I know that you just formatted it, but can you make it right here?)
correct format: `xs="6" sm="6" lg="6"`

File: stories/tooltip/index.jsx [hunk 3]
```diff
@@ -8,28 +8,39 @@ import { WrappedComponent } from '../helpers/wrapped';
...
+                   content={ <><p>Tooltip message will show up here.</p>
+                     <a target='__blank'
+                        href='https://staging.qit.online/sign-in'>Link</a> </> }>TOP</Tooltip>
+        </Col>
       </Row>
-      <Row style={{ justifyContent: 'space-around' }}>
-        <Col s="2" m="4" l="3">
-          <Tooltip content="Tooltip message will show up here" position="bottom">
+      <Row style={ { justifyContent: 'space-around' } }>
+        <Col s='6' m='6' l='6'>
```
Comment (lazzystep) [inline stories/tooltip/index.jsx:30]: wrong breakpoints (I know that you just formatted it, but can you make it right here?)
correct format: `xs="6" sm="6" lg="6"`

File: stories/tooltip/index.jsx [hunk 4]
```diff
@@ -8,28 +8,39 @@ import { WrappedComponent } from '../helpers/wrapped';
...
+          <Tooltip arrow content='Tooltip message will show up here'
+                   position='bottom'>
             BOTTOM
           </Tooltip>
         </Col>
-        <Col s="2" m="4" l="3">
-          <Tooltip content="Tooltip message will show up here" position="left">
+        <Col s='6' m='6' l='6' style={{marginLeft: '1rem'}}>
```
Comment (lazzystep) [inline stories/tooltip/index.jsx:36]: wrong breakpoints (I know that you just formatted it, but can you make it right here?)
correct format: `xs="6" sm="6" lg="6"`

**Review by lazzystep — CHANGES_REQUESTED** [review]
you need to add couple fixes and good to go

**Review by lazzystep — APPROVED** [review]

**Review by vozbrann — APPROVED** [review]

**Review by sergdeev — APPROVED** [review]

**Pull request merged**